File: topicos/poo/node/ex1/Leitor.js
export default class Leitor {
  idade;
  nome;
  livro;


  constructor(idade, nome, livro) {
    this.idade = idade;
    this.nome = nome;
    this.livro = livro;
 
    this.livro.incluirLeitor(this);
  }

  oqueEstaLendo() {
    console.log(`${this.nome} esta lendo ${this.livro.titulo} escrito por ${this.livro.autor}`);
  }
}

File: topicos/poo/java/ex1/Leitor.java
public class Leitor {
  int idade;
  String nome;
  Livro livro;

  public Leitor(int idade, String nome, Livro livro) {
    this.idade = idade;
    this.nome = nome;
    this.livro = livro;
  }

  public void oqueEstalendo() {
    System.out.println(
        this.nome + " esta lendo " +
            this.livro.titulo + " escrito por " +
            this.livro.autor);
  }
}
